js/app/src/components/TimeSeries/TimeSeries.svelte

```diff
@@ -42,7 +42,6 @@
 	export let label: string;
 	export let show_label: boolean;
 	export let colors: Array<string>;
-	export let container: boolean = false;
```

Comment:
In some of these files we delete the `container` prop (like in this one) but in others we keep the prop but don't pass it to the component (like video). I think we should remove the prop entirely from all the front-end components (except the form ones you laid out).

BTW, why only Dropdown, Number, Textbox and not all the form components? Just wondering

Reply:
Restored container everywhere.
Textbox, Dropdown, and Number have slightly container behavour in that instead on the block border disappearing, the input border element disappears. Makes it easier to play with gr.Group.